torch/_inductor/codegen/cpp_utils.py

```diff
@@ -40,7 +40,7 @@
     torch.uint8: "uint8_t",
     torch.bool: "bool",
     torch.bfloat16: "bfloat16",
-    torch.complex64: "complex64",
+    torch.complex64: "std::complex<float>",
```

Comment:
This should be `c10::complex<float>`?

Reply:
I think both works, I changed to `c10::complex<float>` now.